[18] Citation info and basic citation / knowledge flows

jaklinger opened the pull request:
Closes #18 

Collect citation info for AI + covid papers and performs cross-tab by field of study.

jaklinger committed:
tidied notebook

jaklinger committed:
tidied notebook

jaklinger committed:
swapped axis labels

Juan-Mateos commented on cord19/transformers/utils.py:
```diff
@@ -0,0 +1,19 @@
+    return create_engine(url, **engine_kwargs)
+
+
+def contains_keyword(text, keywords=('SARS-CoV-2', 'COVID-19', 'coronavirus')):
```
I would add the keywords used to identify covid papers to the `model_config.yaml` in the project directory file so that we can control them from there.

Juan-Mateos reviewed (commented):
I have a couple a comments about the notebook but the best moment to address them is when we refactor the code in the notebook into a script (future issue?). For now I suggest merging.

* If you `%run ../notebook_preamble.py` at the top of your notebook this takes care of your basic imports and logging and creates a `project_dir` variable you can use throughout.

* I couldn't reproduce the notebook because of hardcoded paths to `sql` credentials. In other projects we are putting those in an `.env` file and loading them with functions from the `dotenv` package.

* Same applies to the MAK API key

* I changed the query to MAK in order to obtain citation information about all papers, not just the AI ones. This worked well, thank you!

I implemented my changes in a copy of the notebook that I will include in a new PR.

Commit:
Modified notebook to collect citation data

jaklinger committed:
removed mag keys, updated notebook from juans feedback

jaklinger committed:
removed mag keys, updated notebook from juans feedback

jaklinger commented:
Great, thanks Juan. 

Regarding the notebooks:

- I've updated my notebook accordingly, so others can use it (using `dotenv` and the `ipy` preamble)
- I've removed the API key from your new notebook so someone doesn't come across it and use it!

jaklinger committed:
migrated keywords to model config

jaklinger commented:
... and now I've migrated the keywords to `model_config.yaml` as requested (thanks, good idea):

https://github.com/nestauk/covid-19_research/pull/21/files#diff-c839db98b6229c56edb5166e3f6027c7R18
